src/Core/RulesConfigurableEventHandlerInterface.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\rules\Core\RulesConfigurableEventHandlerInterface.
+ */
+
+namespace Drupal\rules\Core;
+
+use Drupal\Component\Plugin\ConfigurablePluginInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Symfony\Component\EventDispatcher\Event;
+
+/**
+ * Interface for handling configurable rules events.
+ *
+ * Configurable events have a custom event suffix, which gets appended to the
+ * base event name (= the plugin id), forming the fully-qualified event name,
+ * which is used for triggering reaction rules.
+ *
+ * For example, the fully-qualified event name of an event for viewing an
+ * article node would be "node_view--article", whereas "node_view" is the base
```

Comment:
The event for viewing a node is "rules_entity_view:node", see rules_entity_view().